iib/web/config.py

```diff
@@ -27,7 +27,7 @@ class Config(object):
     IIB_REQUEST_LOGS_DIR: Optional[str] = None
     IIB_REQUEST_RELATED_BUNDLES_DIR: Optional[str] = None
     IIB_REQUEST_RECURSIVE_RELATED_BUNDLES_DIR: Optional[str] = None
-    IIB_USER_TO_QUEUE: Dict[str, str] = {}
+    IIB_USER_TO_QUEUE: Dict[str, Dict[str, str]] = {}
```

Comment:
can we please update the format in README as well, and have a test for queue selection?

Reply:
I am still thinking about keeping backward compatibility. For example by keeping this variable and creating new one with condition only one of them can be set.

Reply:
I changed the code so it accepts both and should not break anything. 
Unit tests for this change are on a way.